lightningd/test/run-log_filter.c

```diff
@@ -7,6 +7,13 @@
 #define fwrite test_fwrite
 static size_t test_fwrite(const void *ptr, size_t size, size_t nmemb,
 			  FILE *stream);
+/* FIXME(vincenzo): Valgrind is reporting a peculiar error during the emission of a notification,
+ * which I cannot pinpoint the cause of. This workaround bypasses the section of the code
+ * responsible for emitting notifications in the logger. It also avoids the need to add
+ * a new argument to the logv function solely for the purpose of skipping code that
+ * only causes issues during testing. */
+#define NO_NOTIFICATION
+
```

Comment:
This needs to be addressed before we can merge this PR. We don't want to have the tested version deviate from the built releases.

Reply:
>We don't want to have the tested version deviate from the built releases.

What do you mean? 

However, I am not sure that I can get rid of it because the test is generating the log line and then an no free memory error happens with valgrid, I will look more into it btw tomorrow